From d7a78c5157bd276a23727a0e4628a5b722bd8c7f Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Thu, 9 Mar 2017 17:50:49 +0200
Subject: [PATCH 1557/5242] MLK-14401: ARM: dts: Add supply for bypassed vddpu

commit  e58796687d6c9ab30d784f3008491650fe6ddf5a from
https://source.codeaurora.org/external/imx/linux-imx.git

Newer kernel versions output a lot of warnings if they encounter a
bypassed regulator without an explicitly configured supply. Like this:

vddpu: bypassed regulator has no supply!
vddpu: bypassed regulator has no supply!
vddpu: bypassed regulator has no supply!

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6dqscm-qwks-rev2.dtsi |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6dqscm-qwks-rev2.dtsi b/arch/arm/boot/dts/imx6dqscm-qwks-rev2.dtsi
index 0dd82ea..f631e66 100644
--- a/arch/arm/boot/dts/imx6dqscm-qwks-rev2.dtsi
+++ b/arch/arm/boot/dts/imx6dqscm-qwks-rev2.dtsi
@@ -142,6 +142,11 @@
 	regulator-allow-bypass;
 };
 
+&reg_pu {
+	vin-supply = <&sw1c_reg>;
+	regulator-allow-bypass;
+};
+
 &clks {
 	assigned-clocks = <&clks IMX6QDL_CLK_LDB_DI0_SEL>,
 			  <&clks IMX6QDL_CLK_LDB_DI1_SEL>;
-- 
1.7.9.5

